### PR TITLE
Feature50/Add-manifest.json

### PR DESCRIPTION
**Pull request by bryandino673**

### CODE REVIEW

File: power-pay-frontend/src/components/okpage.tsx
Comment (stephane-segning) [inline power-pay-frontend/src/components/okpage.tsx:1]: What is this file doing here in the `add-manifest.json` PR?

**Review by stephane-segning — COMMENTED** [review]
Clarify me on this please

**Comment by stephane-segning** [comment]
@bryandino673 What is the difference between this and #119?

File: power-pay-frontend/src/components/okpage.tsx
Comment (Motouom) [reply power-pay-frontend/src/components/okpage.tsx:1]: @bryandino673  consider this comment that Sir @stephane-segning  made please

**Pull request closed without merging**